src/sql/src/plan/statement/ddl.rs

```diff
@@ -775,15 +775,30 @@ fn kafka_sink_builder(
         .key_writer_schema()
         .map(|key_schema| key_schema.canonical_form());
 
-    // Use the user supplied value for replication factor, or default to 1
+    // Use the user supplied value for partition count, or default to -1 (broker default)
+    let partition_count = match with_options.remove("partition_count") {
+        None => -1,
+        Some(Value::Number(n)) => n.parse::<i32>()?,
+        Some(_) => bail!("partition count for sink topics has to be an integer"),
```

Comment:
Minor nit: we more often use "must be" or "should be", instead of "has to be". Don't bother changing it if it means you have to go through CI again, though.

Reply:
It's fine, I fixed it. Github's auto-merge is a god send for these kind of things, no need to monitor the PR :)